react/Section/Section.js

```diff
@@ -21,5 +47,16 @@ export default function Section({ children, className, header, ...restProps }) {
 Section.propTypes = {
   children: PropTypes.node.isRequired,
   className: PropTypes.string,
-  header: PropTypes.bool
+  header: PropTypes.bool,
+  pullout: PropTypes.bool,
+  slim: PropTypes.bool,
+  type: PropTypes.oneOf([TYPE.POSITIVE, TYPE.INFO, TYPE.CRITICAL, TYPE.HELP]),
```

Comment:
Should we refer to this as "tone" instead, since that's the terminology we've already been using? e.g. http://seek-oss.github.io/seek-style-guide/typography